README.md

```diff
@@ -36,7 +36,8 @@ compiler and Gleam dependencies:
       app: @app,
       # ...
       archives: [mix_gleam: "~> 0.4.0"],
-      aliases: MixGleam.add_aliases(aliases()),
+      compilers: [:gleam | Mix.compilers()],
```

Comment:
We should put a note here about how to integrate with Phoenix, which looks to use `aliases()` for its own aliases. See #16.

Reply:
So for Phoenix the function is a private helper in `mix.exs` defined at the bottom on that file. What I usually do and see in other people's projects is that new aliases are being added to that helper function. Seeing the instructions from this PR I personally would know what to do, just by following the `aliases()` function to its definition, and I would paste the alias there. Maybe something like a comment with `# or add this alias to your aliases() function` would suffice?